### DISCUSSION TITLE
what does formatted mean

### DISCUSSION THREAD

**Discussion by akjalok** [fineanmol/Hacktoberfest2021#916] [Q&A] [answered]
I am a newbie and I saw that my project was formatted, can someone guide me to what does it mean?

**Comment by fineanmol** [comment, marked as answer]
People do many contributions to this project, Some Contributions end up with lots of errors and a distorted Layout of the webpage. So by using a working backup point, we restore the project to the last working state & it is a formatted commit.

**Reply by akjalok** [reply]
thanks for helping out

**Reply by fineanmol** [reply]
You are welcome
